### ISSUE TITLE
AutoComplete | clear input field if showClear="true"

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primevue#5443]
https://github.com/primefaces/primeng/issues/15078

**Comment by tugcekucukoglu** [comment]
PrimeVue AutoComplete does not have `showClear` property.

**Comment by nikkorejz** [comment]
Damn the day I decided to use PrimeVue
